siphon/simplewebservice/mesowest.py

```diff
@@ -0,0 +1,124 @@
+import warnings
+
+from bs4 import BeautifulSoup
+import numpy as np
+import pandas as pd
+
+from ..http_util import HTTPEndPoint
+
+warnings.simplefilter("ignore")
```

Comment:
What warnings are we needing to silence?

Reply:
The warning that is being filtered is SettingWithCopyWarning. It is a consequence of rewriting the DataFrame Series slices into their proper type in the loop starting at line 74.